Remote run (#31)
ipfs/benchmarks: README.md

```

> ### Run sub tests

Reviewer: Can you please add a quick description of the difference of a testclass vs a subtest?

Author: We are removing Testcalss and Subtests.

> 

Reviewer: ```suggestion
#### `TESTCLASS`
```

> TESTCLASS:  Use env vairable TESTCLASS to run test just agianst that class.  Options of TestClass are define in the config.

Reviewer: ```suggestion
Use env variable `TESTCLASS` to run test just against that class.  Options of `TESTCLASS` are defined in the config.
```

^ You should also add a link to the config.

```bash
> TESTCLASS="smallFile" node local-add
```

> 

Reviewer: ```suggestion
#### SUBSTEST
```
